en/middleware/micrortps.md

```diff
@@ -145,7 +145,7 @@ The PX4 Autopilot firmware initialization code may in future automatically start
 For example, in order to run the *Client* daemon with SITL connecting to the Agent via UDP:
 
 ```sh
-micrortps_client start -t UDP
+micrortps_agent start -t UDP
```

Comment:
Hi @maxpolzin 

This isn't correct. The section is definitely about building the client, and the client is what is on PX4. The agent should be built on your companion computer or whatever.

If it is not present then there is some other problem though. Can you raise as a discussion board question. My assumption would be that you haven't built with rtps included.